crates/core/transaction/src/plan/action.rs

```diff
@@ -74,6 +77,55 @@ pub enum ActionPlan {
 }
 
 impl ActionPlan {
+    /// Builds a planned [`Action`] specified by
+    /// the [`ActionPlan`].
+    /// Arguments:
+    ///     &self: `ActionPlan`
+    ///     fvk: `FullViewingKey`
+    ///     witness_data: `&WitnessData`
+    ///     memo_key: `Option<PayloadKey>`
+    /// Returns: `Action`
```

Comment:
nit: it's better to leave these out, because Rustdoc will render the arguments based on the actual Rust code, and these will get mashed together with the rest of the markdown.

Reply:
removed